snuba/datasets/entities/discover.py

```diff
@@ -284,6 +284,7 @@ def attempt_map(
                 ]
             ),
         ),
+        ("app_start_type", String(Modifiers(nullable=True))),
```

Comment:
Can you add this to the transactions entity as well please? https://github.com/getsentry/snuba/blob/master/snuba/datasets/entities/transactions.py

Reply:
Yes, I added it, but I'm unsure if I did it correctly. Why is `transaction_source` not in the transaction entity?

Reply:
I added a parameterized test.